modules/apps/27-interchain-accounts/controller/keeper/handshake_test.go

```diff
@@ -111,6 +111,16 @@ func (suite *KeeperTestSuite) TestOnChanOpenInit() {
 			"channel is already active",
 			func() {
 				suite.chainA.GetSimApp().ICAControllerKeeper.SetActiveChannelID(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
+
+				counterparty := channeltypes.NewCounterparty(path.EndpointB.ChannelConfig.PortID, path.EndpointB.ChannelID)
+				channel := channeltypes.Channel{
+					State:          channeltypes.OPEN,
+					Ordering:       channeltypes.ORDERED,
+					Counterparty:   counterparty,
+					ConnectionHops: []string{path.EndpointA.ConnectionID},
+					Version:        string("version"),
```

Comment:
You could use the `TestVersion` variable here

Reply:
good idea